packages/snaps-controllers/src/snaps/SnapController.ts

```diff
@@ -2987,11 +2996,21 @@ export class SnapController extends BaseController<
 
       this.#transition(snapId, SnapStatusEvents.Update);
 
+      const preinstalledArgs =
+        this.#featureFlags.forcePreinstalledSnaps && isLocalSnapId(snapId)
+          ? {
+              preinstalled: true,
+              hideSnapBranding: false,
```

Comment:
Most preinstalled Snaps hide the branding so maybe worth it to set to true to mimic production as closely as possible